spec/controllers/idv/by_mail/enter_code_controller_spec.rb

```diff
@@ -189,7 +189,7 @@
           errors: {},
           pending_in_person_enrollment: false,
           fraud_check_failed: false,
-          enqueued_at: user.pending_profile.gpo_confirmation_codes.last.code_sent_at,
+          enqueued_at: pending_profile.gpo_confirmation_codes.last.code_sent_at,
```

Comment:
These EnterCodeController specs were the only ones that failed as a result of this change. They were dependent on the (now active) profile having been cached as the pending_profile